cargo-culture-kit/src/rules/builds_cleanly_without_warnings_or_errors.rs

```diff
@@ -56,7 +56,7 @@ impl Rule for BuildsCleanlyWithoutWarningsOrErrors {
         build_cmd.arg("build");
         build_cmd
             .arg("--manifest-path")
-            .arg(cargo_manifest_file_path);
+            .arg(context.cargo_manifest_file_path);
```

Comment:
I think `context.` here (and below) is unnecessary, as you've locally bound `cargo_manifest_path` at line 40

Reply:
Good point, will fix.